llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -56,6 +56,7 @@
 #include "llvm/Transforms/Vectorize/LoopVectorize.h"
 #include "LoopVectorizationPlanner.h"
 #include "VPRecipeBuilder.h"
+#include "VPUtils.h"
```

Comment:
```suggestion
#include "VPlanUtils.h"
```

for consistency with other file names in `Vectorize` and avoid confusion with vector predication?